pkg/view/response.go

```diff
@@ -29,7 +29,11 @@ type ErrorResponse struct {
 }
 
 type ResponseString struct {
-	Data string `json:"data"`
+	Data MessageResponse `json:"data"`
+}
+
+type MessageResponse struct {
+	Message string `json:"message"`
```

Comment:
We need to add the project data created into the response body.

Reply:
ok